pkg/reconciler/persistentvolumeclaim/resources.go

```diff
@@ -66,14 +66,21 @@ external:
 }
 
 // isResizing returns true if PersistentVolumeClaimResizing condition is present
+// and not PersistentVolumeClaimFileSystemResizePending
 func isResizing(pvc corev1.PersistentVolumeClaim) bool {
```

Comment:
I don't know if `isResizing` is still the correct domain word, given that clashes with the Resizing condition. 
What we are expressing now is a broader concept